skbio/stats/distance/_base.sublime-workspace

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
Don't forget this.  We don't want project files accidentally committed in the main repository.